<launch>
  <arg name="use_sim_time" default="true" />
  <arg name="gui" default="false" />
  <arg name="headless" default="false" />
  <arg name="world_name" default="$(find jackal_helper)/worlds/BARN/world_1.world" />
  <arg name="rl_config" default="0" />
  <arg name="config_folder" default="initial_configs" />

  <!-- Launch Gazebo with the specified world -->
  <include file="$(find jackal_helper)/launch/gazebo_launch.launch">
<!--    <arg name="debug" value="0" />-->
    <arg name="gui" value="$(arg gui)" />
    <arg name="use_sim_time" value="$(arg use_sim_time)" />
    <arg name="headless" value="$(arg headless)" />
    <arg name="world_name" value="$(arg world_name)" />
      <arg name="front_laser" value="true" />
  </include>

  <node pkg="gatecq" type="planner_instance.py" name="planner_instance" args="$(arg rl_config) $(arg config_folder)" output="screen" required="true"/>

</launch>
